doctoshotgun.py

```diff
@@ -347,7 +331,10 @@ def normalize(cls, string):
             [c for c in nfkd if not unicodedata.combining(c)])
         normalized = re.sub(r'\W', '-', normalized)
         return normalized.lower()
-
+    
```

Comment:
Please run [Prettier](https://prettier.io/) before committing so we keep a consistent code format.